### PR TITLE
update CRD documentation

### LINKED ISSUE #1903: The CRD docs out of date
**Describe the version**
version about:
spiderpool `v0.6.0-rc0`

**Describe the bug**
The docs of the CRD specification are already out of date, it's still v1beta1 version definition.

https://github.com/spidernet-io/spiderpool/blob/v0.6.0-rc0/docs/concepts/spiderippool.md

### PR DESCRIPTION
**Pull request by Icarus9913**
1. update outdated CRD document
2. optimize SpiderMultusConfig CRD (the original field is long to be hard for writing)

Signed-off-by: Icarus9913 [icaruswu66@qq.com](mailto:icaruswu66@qq.com)

**What this PR does / why we need it**:
update docs

**Which issue(s) this PR fixes**:
close #1903

### CODE REVIEW

File: docs/concepts/spiderippool.md [hunk 1]
```diff
@@ -27,42 +27,48 @@ and some other data to the IPPool object for allocation:
 ```text
 // IPPoolSpec defines the desired state of SpiderIPPool
 type IPPoolSpec struct {
-    // specify the IPPool's IP version
+    // specify the IPPool's IP version, Optional
```
Comment (windsonsea) [inline docs/concepts/spiderippool.md:30]:
```suggestion
    // specify the IPPool's IP version, optional
```

File: docs/concepts/spiderippool.md [hunk 2]
```diff
@@ -27,42 +27,48 @@ and some other data to the IPPool object for allocation:
...
     IPVersion *int64 `json:"ipVersion,omitempty"`
 
-    // specify the IPPool's subnet
+    // specify the IPPool's subnet, Required
```
Comment (windsonsea) [inline docs/concepts/spiderippool.md:33]:
```suggestion
    // specify the IPPool's subnet, required
```

File: pkg/k8s/apis/spiderpool.spidernet.io/v2beta1/spidermultus_types.go [hunk 3]
```diff
@@ -61,14 +61,14 @@ type SpiderMacvlanCniConfig struct {
 
 	// +kubebuilder:validation:Optional
 	// +kubebuilder:validation:Minimum=0
-	// +kubebuilder:validation:Maximum=4094
+	// +kubebuilder:validation:Maximum=4095
```
[the remark this answers is not in the document]
Comment (Icarus9913) [reply pkg/k8s/apis/spiderpool.spidernet.io/v2beta1/spidermultus_types.go:64]: SpiderIPPools中的vlan范围是否需要一起更改？

https://github.com/spidernet-io/spiderpool/blob/d3215e93caaef6c80dd4fa15d945109a8fae2b3d/pkg/k8s/apis/spiderpool.spidernet.io/v2beta1/spiderippool_types.go#L28-L32

[the remark this answers is not in the document]
Comment (weizhoublue) [reply pkg/k8s/apis/spiderpool.spidernet.io/v2beta1/spidermultus_types.go:64]: 都是 0-4094

**Pull request merged**